main.go

```diff
@@ -46,7 +48,7 @@ import (
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	"os"
 	client2 "sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
Is the alias `client2` still necessary?

Reply:
I guess, since we have the client defined here - https://github.com/opendatahub-io/opendatahub-operator/blob/8a5b9e7979b3d6674fcecfbcf01aa2f87fdcaee2/main.go#L200-L201.

Maybe I can rename the `client` variable?

Reply:
Updated